src/_common/realm/AppRealmFullCard.vue

```diff
@@ -95,15 +103,14 @@ async function onClick(event: Event) {
 	<div
 		class="-card"
 		:class="{
-			'-hoverable': to || followOnClick,
```

Comment:
in general i think follow on click doesnt make sense even if the entire thing is linkified. i would expect to go to the realm if its a cursor point while hovering over the image, not follow the user. imo it would only make sense to expand the click target to the entire space below the image in the sheet but not more than that.

regardless, i'll revert it to what it was since that's how we had it. just think its odd.